util/src/length_prefix_encoding/encoder.rs

```diff
@@ -92,6 +92,90 @@ pub struct WriteToIoReturn {
 }
 
 #[derive(Clone, Copy, Debug)]
+/// Module description goes here (skipped since it already exists in #479)
```

Comment:
Looks good for now! Lets use the coverage checker to make sure most of the functions from this file are covered.

Can you create a camera ready version and then write a list of possible additions in the comment in the PR?

Reply:
Coverage should be reasonably close to 100% now. The tool doesn't detect docstring tests and the final report doesn't show full coverage (78% for me), despite all functions being covered:

![image](https://github.com/user-attachments/assets/c2e46826-22b8-4a6f-a68c-9388f861ee36)

The coverage also improves when running only the docstring tests vs. running the entire test suite:

![image](https://github.com/user-attachments/assets/42118ea4-da76-4540-8482-8c1afa4d5cea)

Since I've spent way too much time on trying to get more coverage already, I hope it will be enough.